gemfiles/rails_6.1.gemfile.lock

```diff
@@ -0,0 +1,232 @@
+PATH
```

Comment:
Similarly to `Gemfile.lock`, we probably shouldn't commit the `*.lock` files, so as to ensure tests run against latest versions of deps that meet specified constraints, so it matches up with whatever dependency versions get used when someone adds authie to their project.